client/src/components/App.js

```diff
@@ -6,7 +6,10 @@ import Home from './home-page/Home';
 import Cart from './cart-page/Cart';
 import Item from './item-page/Item';
 import Error from './error-page/Error';
+import SignIn from './signin-page/SignIn';
+import OrderConfirmation from './oder-confirmation-page/OrderConfirmation';
```

Comment:
@n4dz 
Look like there is a little typo. ('./oder-...') We should read './order-...'
Please fix file name also

Reply:
Thanks, I changed it 👍